### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by Robbe7730**
It still needs some work visually, but functionally it seems to work fine.

### CODE REVIEW

File: planetwars/client/webpack/app/app.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```
Comment (wschella) [inline planetwars/client/webpack/app/app.html]: Is this file necessary? What does it do?

Comment (Robbe7730) [reply planetwars/client/webpack/app/app.html]: No idea where that came from, sorry.
